group-coordinator/src/main/resources/common/message/OffsetCommitValue.json

```diff
@@ -14,7 +14,8 @@
 // limitations under the License.
 
 {
-  "type": "data",
+  "apiKey": 1,
+  "type": "coordinator-value",
```

Comment:
This is clearly a slightly compromised area because of the way that OffsetCommit has already used two key version numbers. I would say that OffsetCommitValueV0 would have valid version 0, and OffsetCommitValue would have valid versions 1-4. In practice, you'll never find an instance of v0 for OffsetCommitValue because it would actually be OffsetCommitValueV0.

Reply:
OffsetCommitValueV0 has definitely only version 0. For OffsetCommitValue, I think that keeping 0+ does not hurt just in case.